files/src/main/java/com/afollestad/materialdialogs/files/DialogFileChooserExt.kt

```diff
@@ -88,6 +90,10 @@ fun MaterialDialog.fileChooser(
     }
   }
 
+  check(initialDirectory != null) {
```

Comment:
It'd be better to make the parameter non-nullable since it has a default value anyways 🙂

Reply:
The problem here is the following,  the default initialization of:
```java
initialDirectory: File? = context.getExternalFilesDir(),
```
Returns an optional for the reasons I attached above, and I considered necessary to add a property check in case the user don't specify any `initialDirectory` and `context.getExternalFilesDir()` return null. Following like other permission checks on this class. 

Moreover, this is used to guaranty the value for `initialDirectory` used for initialization of the class `FileChooseAdapter` some line below.